crates/re_data_source/src/load_file.rs

```diff
@@ -137,10 +137,17 @@ pub(crate) fn load(
     is_dir: bool,
     contents: Option<std::borrow::Cow<'_, [u8]>>,
 ) -> Result<std::sync::mpsc::Receiver<LoadedData>, DataLoaderError> {
+    #[cfg(target_arch = "wasm32")]
+    let no_external_loaders = true;
+    #[cfg(not(target_arch = "wasm32"))]
+    let no_external_loaders = crate::data_loader::EXTERNAL_LOADER_PATHS.is_empty();
```

Comment:
don't put negations in names

```suggestion
    #[cfg(target_arch = "wasm32")]
    let has_external_loaders = false;
    #[cfg(not(target_arch = "wasm32"))]
    let has_external_loaders = !crate::data_loader::EXTERNAL_LOADER_PATHS.is_empty();
```